mcp/providers.tf

```diff
@@ -0,0 +1,20 @@
+terraform {
+  required_providers {
+    kubernetes = {
+      source = "hashicorp/kubernetes"
+      version = "2.0.2"
```

Comment:
If possible favour some ranges which we know work and/or excluding ones we know don't because this ends up being quite restrictive.